### DISCUSSION TITLE
Should I import the default Way or the "direct" way?

### DISCUSSION THREAD

**Discussion by Gavaii** [carbon-design-system/carbon#14713] [Help]
I found that this import ->
`import OverflowMenuItem from "@carbon/react/lib/components/OverflowMenuItem/OverflowMenuItem";
`
 supports autocompletion and is typed in some components. If available can i use that or should i use the default way-> 
`import { OverflowMenuItem} from "@carbon/react";
` ??

**Comment by tay1orjones** [comment]
Typings should be exposed through the normal/named import, if available.

There have been a couple cases where someone forgot to include the typings in the entrypoint for that component and so, as you found, only the direct path to the file supports autocompletion and other intellisense powered by the component types.

If something has types but isn't working properly, please open an issue or a PR to address it. Thank you!
